enterprise/frontend/src/embedding-sdk/README.md

```diff
@@ -76,24 +76,12 @@ Prerequisites:
 
 Currently, the SDK only works with Metabase version 50.
 
-You have the following options:
-
-### 1. Running on Docker
+### Running on Docker
```

Comment:
Remove the leading number since it's the only option for running Metabase for SDK v1.52.x